pkgs/development/libraries/openxr-loader/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, cmake, python3, libX11, libXxf86vm, libXrandr }:
+
+stdenv.mkDerivation rec {
+  name = "openxr-loader-${version}";
```

Comment:
Please consider using ``pname``.

Reply:
Didn't know we had that for `mkDerivation`, nice!